servant/src/Servant/API.hs

```diff
@@ -83,6 +87,7 @@ import           Servant.API.ResponseHeaders (AddHeader (addHeader),
                                               HList (..), Headers (..),
                                               getHeadersHList, getResponse)
 import           Servant.API.Sub             ((:>))
+import           Servant.API.Times           (FTime(..), toProxy, getFormat, renderTime, parseTime)
```

Comment:
it could also have a nicer name, like `formatProxy` (which is less likely to conflict with any other packages).